test/algorithms/ground_state_solvers/minimum_eigensolver_factories/test_vqe_uvcc_factory.py

```diff
@@ -48,13 +48,7 @@ def setUp(self):
     def test_setters_getters(self):
         """Test Getter/Setter"""
 
-        with self.subTest("Quantum Instance"):
-            self.assertEqual(self._vqe_uvcc_factory.quantum_instance, self.quantum_instance)
-            self._vqe_uvcc_factory.quantum_instance = None
-            self.assertEqual(self._vqe_uvcc_factory.quantum_instance, None)
-
```

Comment:
Was there a reason to remove this getter/setter test completely - the getter and setter still exist right. I can see that setting it to None is arguably an invalid test since the typehint does not include Optional. But a different QuanumInstance could have been created here and then set and checked via the getter similar to what is done for optimizer.